specification/communication/data-plane/Sms/preview/2024-12-10-preview/communicationservicessms.json

```
@@ -62,6 +62,138 @@
           }
         }
       }
+    },
+    "/sms/optouts/:add": {
+      "post": {
+        "tags": [
+          "OptOutsAdd"
+        ],
+        "summary": "Allows Contoso's users to opt out (unsubscribe) from a phone number and block receiving messages from that number.",
```

Let's avoid the term "Contoso" in public docs.

```suggestion
        "summary": "Add phone numbers to the optouts list which shall stop receiving messages from a sender number.",
```